OpenDreamRuntime/DreamManager.Connections.cs

```diff
@@ -32,6 +50,119 @@ public sealed partial class DreamManager {
             _netManager.RegisterNetMessage<MsgLoadInterface>();
             _netManager.RegisterNetMessage<MsgAckLoadInterface>(RxAckLoadInterface);
             _netManager.RegisterNetMessage<MsgSound>();
+
+            var worldTopicAddress = new IPEndPoint(IPAddress.Loopback, _netManager.Port);
```

Comment:
Won't this only listen on localhost? For parity, you'll eventually have to implement cross server comms